tests/test_getitem.py

```diff
@@ -47,6 +47,7 @@ def test_list_with_ints_raise(daa: dak.Array) -> None:
 
 
 def test_single_int(daa: dak.Array, caa: ak.Array) -> None:
+    daa.eager_compute_divisions()
```

Comment:
@douglasdavis can you confirm this is a reasonable fix for the changes to implicit `len` computation?

Reply:
We probably want to avoid `eager_compute_divisions` on the fixture since it's long-lived, perhaps add a
```python
daa = copy.copy(daa)
```
above the divisions compute

Reply:
Ooh, lovely idea.